includes/class-freemius.php

```diff
@@ -1042,7 +1125,10 @@ private function get_caller_main_file_and_type() {
 
 					if ( $caller_file_path == fs_normalize_path( realpath( trailingslashit( $themes_dir ) . basename( dirname( $caller_file_path ) ) . '/' . basename( $caller_file_path ) ) ) ) {
 						$module_type           = WP_FS__MODULE_TYPE_THEME;
-						$caller_file_candidate = $caller_file_path;
+						$caller_file_candidate = basename( dirname( $caller_file_path ) ) .

```

Comment:
@fajardoleo I'll add a comment that this calculates the relative path of the theme + also add an inline example in the comment.